doc.go

```diff
@@ -76,6 +76,8 @@ Application Options:
 	                            (default all interfaces port: 8333, testnet:
 	                            18333, signet: 38333)
 	    --logdir=               Directory to log output
+	    --logcompressor=        Compression algorithm to use when rotating logs.
```

Comment:
nit: this now shows `--logcompressor=[gzip,zstd]` with the `choice` added to the CLI flag.

Reply:
Ah, sorry, I just meant that the text in the doc should probably be updated to show the actual output you would now see on the command line. So just add `[gzip,zstd]` to the `doc.go`.